### DISCUSSION TITLE
如果自定义modal关闭时的动画效果？

### DISCUSSION THREAD

**Discussion by yang200258** [ant-design/ant-design#39037] [General]
### Reproduction link

[http://www.nourl.com](http://www.nourl.com)

### Steps to reproduce

点击关闭modal时的动画

### What is expected?

自定义设置动画

### What is actually happening?

无法设置

| Environment | Info |
| --- | --- |
| antd | 4.20.3 |
| React | 18.2.0 |
| System | mac |
| Browser | chrome |

**Comment by atguo** [comment]
同问，我想做一个Modal关闭时，收起到某一个元素的效果，但是现有api没有什么头绪进行实现

**Comment by Jackie-llv** [comment]
点那个链接为啥跳转到了不可描述的网站
